homeassistant/components/systemmonitor/sensor.py

```diff
@@ -583,7 +583,7 @@ def __init__(
     ) -> None:
         """Initialize the sensor."""
         self.entity_description = sensor_description
-        self._attr_name: str = f"{sensor_description.name} {argument}".rstrip()
+        self._attr_translation_placeholders = {"argument": argument.rstrip()}
```

Comment:
I think we should make our translation placeholder descriptive for the translator. Our translators don't have to be developers (or very technical) to translate.

`"Disk free {argument}"` -> I am guessing the location should be here, but I only know this since I saw the issues and the entities system monitor creates
`"IPv4 address {argument}"` -> is argument an IP address? An internet adapter?

Reply:
That's a bit the point. It's different for different sensors so how would one make it more descriptive?

Reply:
Name it `{device}` ? Or `{interface}`?

Reply:
I constructed a solution so it puts a relevant placeholder